crates/ruff/src/rules/pycodestyle/helpers.rs

```diff
@@ -64,14 +66,14 @@ pub(super) fn is_overlong(
     // begins before the limit.
     let last_chunk = chunks.last().unwrap_or(second_chunk);
     if last_chunk.contains("://") {
-        if width - last_chunk.width() <= limit {
+        if width.width() - last_chunk.width() <= limit.width() {
```

Comment:
You could implement `PartialOrd`, and `Ord` to avoid the unwrapping together with implementing `impl Add for Width` and impl `Sub for Width` so that you can write: `if width - last_chunk <= limit`

Reply:
I see. I like that. What do you think of renaming the method to `get` similar to `NonZeroU32::get` or implement `From<Width> for u32` and use `u32::from(width) - u32::from(last_chunk) <= u32::from(limit)`?